backend/src/BuiltinCliHost/Libs/Cli.fs

```diff
@@ -342,6 +342,7 @@ let fns : List<BuiltInFn> =
       deprecated = NotDeprecated }
 
 
+    // TODO: this doesn't seem to be using the new parser
     { name = fn "cliExecuteFunctionWithNewParser" 0
```

Comment:
maybe rename `cliExecuteFunctionWithNewParser` to just `cliExecuteFunction` and remove the other `cliExecuteFunction` function?

Reply:
sounds good